apps/search/src/assets/i18n/en.json

```diff
@@ -1,29 +1,30 @@
 {
-  "records": "records",
-  "recordViewable": "viewable",
-  "recordDownloadable": "downloadable",
-  "readMore": "Read more",
-  "search.loading": "Loading ...",
-  "dropFile": "drop file",
-  "Back": "Back",
-  "search.field.any.placeholder": "Search datasets, services and maps ...",
-  "search.field.sortBy": "Sort by",
-  "relevancy": "Relevancy",
-  "createDate": "New records",
-  "dateStamp": "Last updates",
-  "popularity": "Popularity",
-  "results.records.hits.found": "{hits, plural, =0{No documents match the specified search.} one{} other{{hits} records found.}}",
-  "results.records.hits.found=0.help": "Suggestions: <ul class='list-disc list-inside'><li>Try other words</li><li>Specify fewer words</li></ul>",
-  "results.layout.selectOne": "Results layout",
-  "facets.block.title.tag": "Keywords",
+  "availableInDownloadService": "Download",
+  "availableInViewService": "View",
+  "facets.block.title.OrgForResource": "Organisation",
+  "facets.block.title.availableInServices": "Available for",
   "facets.block.title.cl_hierarchyLevel.key": "Resource type",
-  "facets.block.title.cl_status.key": "Status",
   "facets.block.title.cl_maintenanceAndUpdateFrequency.key": "Update frequency",
   "facets.block.title.cl_spatialRepresentationType.key": "Spatial representation",
+  "facets.block.title.cl_status.key": "Status",
   "facets.block.title.creationYearForResource": "Creation year",
-  "facets.block.title.th_regions_tree.default": "Regions",
-  "facets.block.title.OrgForResource": "Organisation",
   "facets.block.title.resolutionScaleDenominator": "Scale",
+  "facets.block.title.tag": "Keywords",
   "facets.block.title.tag.default": "Tag",
-  "facets.block.title.availableInServices": "Available for"
+  "facets.block.title.th_regions_tree.default": "Regions",
+  "nav.back": "Back",
+  "record.action.download": "downloadable",
+  "record.action.view": "viewable",
+  "record.more.details": "Read more",
+  "records": "records",
+  "results.layout.selectOne": "Results layout",
+  "results.records.hits.found": "{hits, plural, =0{No documents match the specified search.} one{} other{{hits} records found.}}",
+  "results.records.hits.found=0.help": "Suggestions: <ul class='list-disc list-inside'><li>Try other words</li><li>Specify fewer words</li></ul>",
+  "results.sortBy.createDate": "New records",
+  "results.sortBy.dateStamp": "Last updates",
+  "results.sortBy.popularity": "Popularity",
+  "results.sortBy.relevancy": "Relevancy",
+  "search.field.any.placeholder": "Search datasets, services and maps ...",
+  "search.field.sortBy": "Sort by",
+  "search.loading": "Loading ..."
```

Comment:
how did you change the order ? Did you use the npm script to extract the translations ?

Reply:
JSON sort plugin.

> > * Apply translation scheme to english translations
> What does it mean ?

We discussed to apply a common pattern/scheme to translation keys - like in
https://github.com/geonetwork/geonetwork-ui/blob/master/apps/datafeeder/src/assets/i18n/en.json or https://github.com/geonetwork/geonetwork-microservices/blob/main/modules/services/ogc-api-records/service/src/main/resources/messages/api.properties
no?

Reply:
ok, I was confused by the term "schema"